debug/vector-tile-test.css

```diff
@@ -1,7 +1,7 @@
 
 path.fclass { stroke: blue; }
 
-svg.web-map.style-scope path.fclass._1 { stroke: blue; fill: #FF8C00;}
+svg.mapml-web-map.style-scope path.fclass._1 { stroke: blue; fill: #FF8C00;}
```

Comment:
I'm not sure how/where this SVG selector is used (it seems it's not to be in use atm). Should it also change to match the new `mapml-web-map` class, or do I just revert this change?